examples-server/templates/example.njk

```diff
@@ -5,6 +5,9 @@
   <link href="/css/index.css" rel="stylesheet">
 </head>
 <body>
+  <script>
+    document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');
+  </script>
```

Comment:
This was missing: https://frontend.design-system.service.gov.uk/get-started/#5-get-the-javascript-working